sdk/communication/Shared/src/CommunicationRecordedTestSanitizer.cs

```diff
@@ -38,9 +51,9 @@ public override string SanitizeVariable(string variableName, string environmentV
         {
             return variableName switch
```

Comment:
nit: let's convert this method to expression body as well

Reply:
done!